Gems/PhysX/Code/Editor/ColliderCapsuleMode.h

```diff
@@ -28,28 +30,21 @@ namespace PhysX
         void Teardown(const AZ::EntityComponentIdPair& idPair) override;
         void ResetValues(const AZ::EntityComponentIdPair& idPair) override;
 
+        // CapsuleViewportEdit ...
+        const AZ::Transform GetCapsuleWorldTransform() const override;
+        const AZ::Transform GetCapsuleLocalTransform() const override;
+        const AZ::Vector3 GetCapsuleNonUniformScale() const override;
+        const float GetCapsuleRadius() const override;
+        const float GetCapsuleHeight() const override;
+        const void SetCapsuleRadius(float radius) override;
+        const void SetCapsuleHeight(float height) override;
```

Comment:
suggestion: As we're inheriting privately for now these should probably be in the private section of the class that's overriding I think

Reply:
Oh yeah, good call, will move.

Reply:
Moved to the private section.

Reply:
Nice one thanks!